packages/react/src/index.tsx

```diff
@@ -54,13 +82,18 @@ export const useParallel = () => {
     }
   }, [parallel])
 
-  if (!isPromise(promise)) {
+  if (!isPromise(parallelPromise)) {
     console.warn('You must call loadParallel and place the result in a <ParallelProvider parallel={result}> wrapper')
-    return {}
+    return {
+      isLoaded: false,
+    }
   }
 
   // if we haven't loaded, return empty object
-  if (!parallel) return {}
+  if (!parallel)
+    return {
+      isLoaded: false,
```

Comment:
I added this field in so that there's a simple, clear way to [discriminate](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#discriminated-unions) between loaded and unloaded `useParallel`results.